sql/moz-fx-data-shared-prod/telemetry_derived/desktop_eng_agg_v1/query.sql

```diff
@@ -0,0 +1,31 @@
+SELECT
+  submission_date,
+  first_seen_date,
+  attribution_campaign,
+  attribution_content,
+  attribution_dlsource,
+  attribution_medium,
+  attribution_ua,
+  normalized_os,
+  os_version_major,
```

Comment:
to be replaced with normalized_os_version (because otherwise we can't separate Windows 11, they both are version 10 😞

Reply:
Updated, thanks!